packages/clerk-js/src/core/clerk.ts

```diff
@@ -1185,12 +1192,28 @@ export default class Clerk implements ClerkInterface {
     this.#emit();
   };
 
+  /**
+   * @deprecated This method will be dropped in the next major release.
+   * This method is only used in another deprecated part: `invitationList` from useOrganization
+   */
   __unstable__invitationUpdate(invitation: OrganizationInvitationResource) {
+    deprecated(
+      '__unstable__invitationUpdate',
+      'We are completely dropping this method as it was introduced for internal use only',
+    );
     this.#lastOrganizationInvitation = invitation;
     this.#emit();
   }
 
+  /**
+   * @deprecated This method will be dropped in the next major release.
+   * This method is only used in another deprecated part: `membershipList` from useOrganization
+   */
   __unstable__membershipUpdate(membership: OrganizationMembershipResource) {
+    deprecated(
+      '__unstable__invitationUpdate',
```

Comment:
🔧 `__unstable__membershipUpdate`

Reply:
@nikosdouvlis This needed change was missed

Reply:
https://github.com/clerkinc/javascript/pull/1889#event-10662308104